src/components/ImportParameters.jsx

```diff
@@ -0,0 +1,158 @@
+import { useState } from "react";
+
+const parameterDetails = {
+  resource: { 
+    name: <><code>resource</code></>, 
+    type: "IRI, String", 
+    description: <>The file name to write to. 
+      If it contains an extension, this is used to automatically set the <code>compression</code> parameter. 
+      If set to the empty string <code>""</code>, the tuples are read from stdin until pressing <code>Ctrl + D</code>. 
+      This is restricted to one stdin resource per program. If omitted, this is set based on the predicate name, 
+      file format and compression type: <code>&lt;predicate-name&gt;.&lt;format&gt;.&lt;compression&gt;</code>.</> 
```

Comment:
this is the only case where it _might_ be acceptable to have markup, but it should then be used consistently (e.g., `stdin` needs to be wrapped in `<code>` tags as well). The reference to `Ctrl + D` is inappropriate; data is read until end-of-file (which may be triggered by pressing `C-d` on some systems, but this is not universal).